docker-compose.yml

```diff
@@ -183,24 +186,54 @@ services:
     ports:
       - '9221:9221'
 
+  # tools:
```

Comment:
What does the tools container do?

Reply:
Sorry about that... I should remove it.  It was to run the Platform tools API (tapi).  I had been using it a bit for quick 'dummy' account creation, and was going to leave it as placeholder until I got the tools Dockerfile merged into the `platform` repo, though, I now doubt it will happen.

I'll take this out, and remove reference to the tools service in the README.